### PR TITLE
Python challenge submission

### PR DESCRIPTION
**Pull request by niketagrawal**
This repository is a nice tool for teaching and learning purposes.

### CODE REVIEW

File: gizmo.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    def speak(self):
+        print(self.name)
+
+def multiplication_table(zero_out_multiples=None):
+    """Creates a multiplication table.
+
+    Returns a two-dimensional NumPy array that contains the multiplication 
+    table from 1 to 12.  
+    
+    Paramateres
```
Comment (wmvanvliet) [inline gizmo.py]:
```suggestion
    Parameters
```

File: gizmo.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    print(f"Hello {name}, how are things in {country}?")
+
+def spell(word):
+    wordlength = 0
+    letters = []
+    spelling = ""
+    for letter in word:
+        wordlength += 1
+        letters.append(letter)
+    for i in range (wordlength):
```
Comment (wmvanvliet) [inline gizmo.py:14]: You don't need to make a list of characters. You can iterate directly over a string:

```python
for character in word:
```

**Review by wmvanvliet — COMMENTED** [review]
Almost there! Nice work so far.

File: gizmo.py (hunk 1 above)
Comment (niketagrawal) [reply gizmo.py]: Thanks for pointing out the typo! I just did not see it.

File: gizmo.py (hunk 2 above)
Comment (niketagrawal) [reply gizmo.py:14]: Thanks for the tip.

**Comment by wmvanvliet** [comment]
All green, well done!

**Pull request closed without merging**